src/Features/CSharp/Portable/UseCompoundAssignment/Maps.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseCompoundAssignment
+{
+    internal static class Maps
```

Comment:
>Maps [](start = 26, length = 4)

nit: I'm not sure those maps deserve a separate file. Would it be possible to put them in the analyzer?

Reply:
They could be... i didn't feel strongly about it at the time.  It was just convenient to ahve a simple place that both the analyzer/fixer could reference.